### DISCUSSION TITLE
Provide verbose error message when not being able to add database

### DISCUSSION THREAD

**Discussion by SetTrend** [pawelsalawa/sqlitestudio#4339] [Ideas]
I'm unsuccessfully trying to open an existing `*.db3` file on Windows 10x64.

The only response I get is a red exclamation mark and an error message telling me that "the database C:/Users/.../data.db3 could not be added."

![SQLitestudio](https://user-images.githubusercontent.com/9283914/146270291-4608c67c-00b1-4089-aeef-5c4a7d7ece97.png)

I suggest to have SQLitestudio provide a thoughtful error message, stating the technical reason for the failure and probably adding hints on solving the issue.

The exclamation mark icon should come with the proposed thoughtful error message, too, by means of a tooltip.

The current situation isn't useful nor helpful.

**Comment by pawelsalawa** [comment]
Created #4341 for implementation.
